rhobs/alerting/data_plane/prometheus.component_alerts.yaml

```
@@ -25,4 +25,21 @@ spec:
           Component controller in Pod {{ $labels.pod }} for namespace
           {{ $labels.namespace }} on cluster {{ $labels.source_cluster }} is failing to
           successfully delete at least 95% of components over the past hour
+        runbook_url: https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/has/component-delete-failed.md
+    - alert: ComponentCreationErrors
+      expr: |
+        (increase(has_component_failed_creation_total[1h]))
+        /
+        (increase(has_component_creation_total[1h]))  > 0.05
+      for: 5m
+      labels:
+        severity: warning
+        slo: true
```

```suggestion
        slo: "true"
```
There was a recent bug related to this but fixed in #138.

---

@amisstea I've included your suggestion. Thanks :)